kfbot-refactor-phase2/src/main/java/com/twitchbotx/bot/handlers/CountHandler.java

```diff
@@ -195,9 +227,31 @@ public static String handleSubMessage(
             int giftAmount,
             int subPoints,
             boolean prime,
-            String subMonths)
-        {
-            String tier = "";
+            String subMonths
+        ) {
+            String tier = assessTier(subPoints)
+            SubType subType = assessSubType(massGift, singleGift, subMonths, prime);
+            String response = subType.buildMessage(subDisplayName, tier, subMonths, giftRecipient, giftAmount);
+
+            // bundle and send reply to messenger and event list
+            sendEvent(response);
+            return response;
+        }
+
+        private static void sendEvent(final String msg) {
+            String event = msg;
+            
+            Platform.runLater(new Runnable() {
+                @Override
+                public void run() {
+                    guiHandler.bot.getStore().getEventList().addList(event);
+                }
+            });
+        }
+
+        // TODO: Move method higher up
+        // The returned value should be one of the parameters passed in to handleSubMessage
+        private static String assessTier(int subPoints) {
             switch (subPoints) {
```

Comment:
need to create a String tier = ""; and return it.

Reply:
I'll do you one better